index.html

```diff
@@ -0,0 +1,417 @@
+              <p class="features__card-text">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+                diam nonumy.
+              </p>
+            </div>
+          </li>
+          <li class="features__item">
+            <div class="features__card">
+              <img src="./assets/earphone.svg" alt="shield">
+              <h4 class="features__card-heading">
+                Full time customer support - 24/7
+              </h4>
+              <p class="features__card-text">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+                diam nonumy.
+              </p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="about-us">
+      <div class="about-us__header">
+        <h4 class="about-us__label">Why should you work with us?</h4>
+        <h2 class="about-us__heading">
+          To upscale your business to the next level
+        </h2>
+        <p class="about-us__text">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+          erat, sed diam voluptua invidunt ut labore.
+        </p>
+      </div>
+      <div class="about-us__body">
+        <ul class="about-us__list">
+          <li class="about-us__item">
+            <img src="./assets/arrow.svg" alt="arrow">
+            <p class="about-us__item-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </li>
+          <li class="about-us__item">
+            <img src="./assets/arrow.svg" alt="arrow">
+            <p class="about-us__item-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </li>
+          <li class="about-us__item">
+            <img src="./assets/arrow.svg" alt="arrow">
+            <p class="about-us__item-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="cta">
+      <div class="cta__header">
+        <h4 class="cta__label">Why should you work with us?</h4>
+        <h2 class="cta__heading">
+          To upscale your business to the next level
+        </h2>
+        <p class="cta__text">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+          erat, sed diam voluptua invidunt ut labore.
+        </p>
+        <button class="cta__btn">Get Free trial</button>
+      </div>
+      <div class="cta__body">
+        <img src="./assets/stats.png" class="cta__img" alt="stats">
+      </div>
+    </section>
+    <div class="logos">
```

Comment:
Improve this block name it seems like you are relating the logo block

Reply:
its is a block containing a lot of logos

Reply:
and thats the name, they used to name that section in figma